src/main/java/seedu/todo/model/TodoList.java

```diff
@@ -99,50 +99,58 @@ public ImmutableTask add(String title, Consumer<MutableTask> update) throws Vali
         update.accept(validationTask);
         Task task = validationTask.convertToTask();
         tasks.add(task);
-
         saveTodoList();
         return task;
     }
 
     @Override
-    public ImmutableTask delete(int index) throws ValidationException {
-        Task task = tasks.remove(index);
+    public List<ImmutableTask> delete(List<Integer> indexes) throws ValidationException {
+        List<ImmutableTask> tasksRemoved = new ArrayList<>();
+        
+        for (Integer i : indexes) {
```

Comment:
The use of i is generally the convention used when iterating through integers though :x